clang/test/lab1/polozov_vladislav/my_test.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/PluginWarningDeprecated%pluginext -plugin warning-deprecated %s 2>&1 | FileCheck %s --check-prefix=DEPRECATED
+
+// DEPRECATED: warning: find 'deprecated' in function name
+void deprecated_f(int a, int b){
+    int c = a + b;
+}
+
+// DEPRECATED-NOT: warning: find 'deprecated' in function name
+void Matrix_Mult(){
+    ;
+};
+
+// DEPRECATED-NOT: warning: find 'deprecated' in function name
+void Deprecated(int c){
+    int a = c - 10;
+    int b = a + c;
+};
+
+// DEPRECATED: warning: find 'deprecated' in function name
+int function_with_deprecated(int a, int b){
+    return a - b;
```

Comment:
Checks are not really attached to these lines. This test actually checks that there are at least 2 "deprecated" in function name. You need to either split your file checks (see --split-input-file) or come up with other CHECKs